### PR TITLE
Fix jackson version to use valid Jackson version

### PR DESCRIPTION
**Pull request by Grimoren**
## Fixes Issue #
https://github.com/dependency-check/dependency-check-gradle/issues/260
https://github.com/jeremylong/DependencyCheck/issues/4280
## Description of Change
use valid jackson version

## Have test cases been added to cover the new functionality?
N/A: no new functionality.

### CODE REVIEW

**Comment by Grimoren** [comment]
It seems that 2.13.2.1 is valid for most jackson with exception to jackson-bom:
https://mvnrepository.com/artifact/com.fasterxml.jackson/jackson-bom/2.13.2.20220324

**Comment by Janpopan** [comment]
```xml
<dependency>
    <groupId>com.fasterxml.jackson</groupId>
    <artifactId>jackson-bom</artifactId>
    <version>2.13.2.20220324</version>
    <type>pom</type>
</dependency>
```

File: pom.xml [hunk 1]
```diff
@@ -1094,6 +1094,12 @@ Copyright (c) 2012 - Jeremy Long
                 <artifactId>plexus-utils</artifactId>
                 <version>3.4.1</version>
             </dependency>
+            <dependency><!-- Temporary upgrade to fix jackson mess up. Be certain to delete once version is fixed -->
+                <groupId>com.fasterxml.jackson</groupId>
+                <artifactId>jackson-bom</artifactId>
+                <version>2.13.2.20220324</version>
+                <type>pom</type>
+            </dependency>
```
Comment (mprins) [inline pom.xml:1102]: to use a BOM in maven the `<scope>import</scope>` should be used

**Review by juergenzimmermann — CHANGES_REQUESTED** [review]
Meanwhile there is Jackson BOM `2.13.2.20220328`. See https://github.com/FasterXML/jackson-databind/issues/3428

**Comment by chadlwilson** [comment]
According to https://github.com/FasterXML/jackson-databind/issues/3428#issuecomment-1081330160 it seems the right way forward here is to roll forward to `jackson.version` of `2.13.2.2` which corrects the Gradle metadata causing problems for Gradle users (and dependency-check-gradle).

(Or move to a BOM-based approach entirely which would be more work, and involve removing the specific versions for `jackson-databind`)

File: pom.xml (hunk 1 above)
Comment (philsegal) [reply pom.xml:1102]: try using Jackson version `2.13.2.2` instead?

**Comment by jeremylong** [comment]
superseded by #4285

**Pull request closed without merging**